build.sh

```diff
@@ -1,14 +1,27 @@
 #!/bin/bash
 echo "Build Script for eduVPN (and derivatives)"
+
+#Check if HomeBrew Installed
+if ! [ -x "$(command -v brew)" ]; then
+  echo 'Homebrew is not installed. Installing Homebrew.... ' >&2
+  ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+fi
+
+
 # Check if the Carthage is installed
 if ! [ -x "$(command -v carthage)" ]; then
-  echo 'Error: Carthage is not installed.' >&2
+  echo 'Carthage is not installed. Installing Carthage' >&2
+  brew install carthage
   python -mwebbrowser https://github.com/Carthage/Carthage
```

Comment:
The browser does not need to be opened anymore if you use homebrew to install carthage, right?